decidim-comments/app/forms/decidim/comments/comment_form.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+module Decidim
+  module Comments
+    # A form object used to create comments from the graphql api.
+    #
+    class CommentForm < Form
+      attribute :body, String
+
+      mimic :comment
+
+      validates :author, :commentable, :body, presence: true
+
+      attr_reader :author, :commentable
+
+      def initialize(attributes = {})
+        @author = attributes.delete(:author)
+        @commentable = attributes.delete(:commentable)
```

Comment:
I'm OK with having `commentable` and `author`, but they should be form attributes, not `attr_reader`